src/Sylius/Bundle/ApiBundle/Command/Checkout/ChooseShippingMethod.php

```diff
@@ -50,4 +44,19 @@ public function setOrderTokenValue(?string $orderTokenValue): void
     {
         $this->orderTokenValue = $orderTokenValue;
     }
+
+    public function getSubresourceId(): ?string
+    {
+        return $this->shipmentId;
+    }
+
+    public function setSubresourceId(?string $subresourceId): void
+    {
+        $this->shipmentId = $subresourceId;
+    }
+
+    public function getSubresourceIdAttributeKey(): string
+    {
+        return 'shipmentId';
```

Comment:
IMHO, the command should not be aware of path definition details. An additional interface is ok, but I would prefer to make more detailed (get/setShipmentId), make additional transformer, where `shipmentId` will be injected into the constructor. 

Also, can you check if we can use parameters in route definition?

Reply:
I added `getSubresourceIdAttributeKey` instead of details like get/setShipmentId, because i would use it in next cases like payment, out of the box without duplicated boilerplate code.